score/mw/com/impl/tracing/configuration/tracing_filter_config_test.cpp

```diff
@@ -727,5 +756,119 @@ TEST_F(TracingFilterConfigGetNumberOfTracingSlotsDeathTest, ConfigurationContain
     EXPECT_DEATH(tracing_filter_config_.GetNumberOfTracingSlots(configuration_.value()), ".*");
 }
 
+using TracingFilterConfigGetNumberOfTracingSlotsFixture = ConfigurationFixture;
+TEST_F(TracingFilterConfigGetNumberOfTracingSlotsFixture, InsertingNoTracePointsWithTraceDoneCBReturnsZero)
+{
+    const auto trace_point_type_0 = ProxyEventTracePointType::SUBSCRIBE;
+    const auto trace_point_type_1 = ProxyEventTracePointType::UNSUBSCRIBE;
+    const auto trace_point_type_2 = ProxyEventTracePointType::GET_NEW_SAMPLES;
+    const auto trace_point_type_3 = ProxyFieldTracePointType::GET;
+    const auto trace_point_type_4 = ProxyFieldTracePointType::SET;
```

Comment:
These trace point types you can define as constexpr global variables.
Best also put the specific trace point type into the variable name.
That way you can reuse them in the tests without having to redefine them.